### PR TITLE
Adding a simple FDP and EJP-RD beacon implementation configuration  

### PR DESCRIPTION
**Pull request by wna-se**
Addressed #9

### CODE REVIEW

File: config/fair-in-a-box/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    environment:
+      - FDP_MONGO_HOST=mongo
+      - FDP_MONGO_USER=root
+      - FDP_MONGO_PASS=password
+      - FDP_MONGO_PORT=27017
+      - FDP_MONGO_DB=fdp
+      - FDP_TRIPLE_STORE_TYPE=4
+      - FDP_TRIPLE_STORE_URL=http://graphdb:7200
+      - FDP_TRIPLE_STORE_REPOSITORY=fdp
+      - FDP_TRIPLE_STORE_USERNAME=fdp
```
Comment (MalinAhlberg) [inline config/fair-in-a-box/docker-compose.yml:55]: Read credentials form `.env` instead?

File: docs/fair-in-a-box.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+```shell
+docker-compose up -d
+```
+
+Access the services: 
+
+* FAIR Data Point Client at http://localhost:7070 ([documentation](https://fairdatapoint.readthedocs.io/en/latest/))
+* FAIR Data Point Server at http://localhost:7080 ([documentation](https://fairdatapoint.readthedocs.io/en/latest/))
+* GraphDB at http://localhost:7200 ([documentation](https://graphdb.ontotext.com/documentation/10.6/))
+* EJP-RD-Beacon-in-a-Box at http://localhost:8000 ([documentation](https://fairdatapoint.readthedocs.io/en/latest/))
```
Comment (MalinAhlberg) [inline docs/fair-in-a-box.md:22]:
```suggestion
* EJP-RD-Beacon-in-a-Box at http://localhost:8000 ([documentation](https://fairdatapoint.readthedocs.io/en/latest/))

Log in details can be found in the `.env` file.
```
(and what is the password for the fdp?)

File: docs/fair-in-a-box.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# FAIR Data Point and EJP-RD Beacon API implementation
+
+Test in the local configuration folder (currently identical to global)
```
Comment (MalinAhlberg) [inline docs/fair-in-a-box.md:3]: I needed to copy the `.env` file to this folder in order for this to work, maybe that should be stated here.

**Review by MalinAhlberg — APPROVED** [review]
Really nice!

I left some minor comments and questions, but will also approve since everything seems to be working.

File: docs/fair-in-a-box.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+```shell
+docker-compose up -d
+```
+
+Access the services: 
+
+* FAIR Data Point Client at http://localhost:7070 ([documentation](https://fairdatapoint.readthedocs.io/en/latest/))
+* FAIR Data Point Server at http://localhost:7080 ([documentation](https://fairdatapoint.readthedocs.io/en/latest/))
+* GraphDB at http://localhost:7200 ([documentation](https://graphdb.ontotext.com/documentation/10.6/))
```
[the remark this answers is not in the document]
Comment (wna-se) [reply docs/fair-in-a-box.md:21]: Indeed. The docker-compose.override.yml the GraphDB port is exposed on the docker host which (I think) is then usually exposed automatically on localhost of host computer using a redirect.

**Pull request merged**